infra/is-ngcc-necessary.ts

```diff
@@ -22,20 +23,24 @@ for (const packageName of packages) {
 
   const metadataPath = resolve(packagePath, typingsPath.replace(/\.d\.ts$/, '') + '.metadata.json');
 
-  if (!existsSync(metadataPath)) {
+  if (existsSync(metadataPath)) {
+    ngccPackages.push(packagePath);
+  } else {
     nonNgccPackages.push(packagePath);
   }
 }
 
+console.log(chalk.gray(`${ngccPackages.length} out of ${packages.length} packages still need ngcc processing.`));
+
 if (nonNgccPackages.length > 0) {
   console.error(chalk.red('The following packages no longer need to be processed by ngcc. Please add them to `./infra/non-ngcc-packages.json`.'));
   for (const packageName of nonNgccPackages) {
-    console.error(`- ${packageName}`);
+    console.error(chalk.yellow(`- ${packageName}`));
   }
   process.exit(1);
 }
 
-console.error(chalk.gray('No packages that no longer need ngcc processing.'));
+console.log(chalk.gray('No packages that no longer need ngcc processing.'));
```

Comment:
I know you are just changing the log level here, but the double negative of `no packages that no longer` is a but confusing, can we reword that?

Reply:
:+1:

I will remove this altogether, since e already print how many packages need still need ngcc processing in the previous `console.log()` statement.